### DISCUSSION TITLE
Invitation to the Scryer Prolog Meetup 2024 in Vienna

### DISCUSSION THREAD

**Discussion by triska** [trealla-prolog/trealla-prolog#535] [General]
Dear all,

on Nov. 7th and 8th, the **Scryer Prolog Meetup 2024** takes place in Vienna, Austria:

https://www.digitalaustria.gv.at/eng/insights/Digital-Austria-Events-EN/Scryer-Prolog-Meetup-2024.html

Especially since Scryer and Trealla already share a few libraries, and also allow many portable applications due to their shared perspective on conformance to the Prolog ISO&nbsp;standard, I would like to invite all Trealla programmers to also attend the meetup! You are all welcome, and I hope that the meetup fosters further cooperation between these systems! The meetup takes place in an awesome venue, the **oldest hotel in Vienna**, a sight of its own which also hosts interesting historical artefacts and, like Scryer and Trealla, fittingly combines tradition with modernity!

I hope to see many of you in November!

Thank you and all the best,
Markus

**Comment by infradig** [comment]
Sounds delightful. I would urge anyone that can attend to do so. Cooperation can only be helpful. As for me, it would be leaning towards doubtful at this stage.
